src/base/os.lua

```diff
@@ -11,7 +11,6 @@
 --
 
 	premake.override(os, "execute", function(base, cmd)
-		cmd = path.normalize(cmd)

```

Comment:
Yeah no, you can't just remove stuff until the problem goes away. There is probably an issue in path.normalize() in the say it is removing the "../" sequences.